docker-compose.yml

```diff
@@ -41,7 +41,12 @@ services:
   telegraf:
     image: docker.io/library/telegraf:1.29
     volumes:
+      - /:/hostfs:ro
+      - ./telegraf.d:/etc/telegraf/telegraf.d:ro
```

Comment:
@glimchb Do we require this right now we don't have telegraf.d directory with extra plugin right know

Reply:
you are correct, please remove this line

Reply:
Done